Doc/whatsnew/3.7.rst

```diff
@@ -297,6 +297,16 @@ string
 expression pattern for braced placeholders and non-braced placeholders
 separately.  (Contributed by Barry Warsaw in :issue:`1198569`.)
 
+time
+----
+
+The ``time.clock()`` function has been removed, it was deprecated since Python
+3.3.  The function was not portable: on Windows it mesured wall-clock, whereas
```

Comment:
Perhaps replace:

`it was deprecated since Python 3.3.`

with:

`as it has been deprecated since Python 3.3.`